tests/lib.rs

```diff
@@ -607,3 +607,63 @@ fn deploy_fails_with_floats() {
         Err(rusk_vm::VMError::InstrumentalizationError(_))
     ));
 }
+
+#[test]
+fn persistance() {
+    use microkelvin::DiskBackend;
+
+    let counter = Counter::new(99);
```

Comment:
In fact at the end looks like you actually just used `Counter` to test it, so please remove the `Persistance` contract you created that doesn't seems be used anywhere at this point.

Also, I would suggest to slightly modify the test `calling` so that it deploys the three contracts, drop state, and then restore it and call them.